lua/luasnip/loaders/util.lua

```diff
@@ -1,16 +1,11 @@
-local session = require("luasnip.session")
-
 local function filetypelist_to_set(list)
 	vim.validate({ list = { list, "table", true } })
 	if not list then
 		return list
 	end
 	local out = {}
 	for _, ft in ipairs(list) do
-		-- include redirected filetypes.
-		for _, resolved_ft in ipairs(session.ft_redirect[ft]) do
-			out[resolved_ft] = true
-		end
+		out[ft] = true
```

Comment:
I don't think we can ignore the redirects (esp. not here, this is also used by the vscode-loader).

Reply:
The vscode is also fine because it calls util.get_snippet_filetypes() in M._lazy(). Rather, the snippet file loading was duplicated when this was present.

Reply:
Oh you're right, there is some duplication in vscode-lazy-load. AFAICT regular `load({include=some_extended_filetype})` has to load the snippets for the extended filetypes as well, and they are only included here.
But maybe it's best to not resolve redirected filetypes here, but only explicitly.

Reply:
Yes, I think it would be better to explicitly select the snippet file you want to use if you are a user for such purposes.